pom.xml

```diff
@@ -108,10 +108,9 @@ under the License.
     </dependency>
     <dependency>
       <groupId>org.eclipse.aether</groupId>
-      <artifactId>aether-api</artifactId>
+      <artifactId>aether-util</artifactId>
       <!-- the same version as in Maven 3.2.5 -->
```

Comment:
Shouldn't we upgrade to Maven 3.6.5 ?
It could be a separate PR though...

Reply:
yes next PR with separate issue